Week2/solution/solution_2.txt

```diff
@@ -0,0 +1,42 @@
+NOTE ::
+First we need to write (pragma foreign_key = ON)

```

Comment:
why you use it?

Reply:
Is it ok if we don't write this command?

Reply:
Foreign key is not work